src/style.css

```diff
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 400;
+  color: #3d4e61;
+  font-family: avenir, sans-serif;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+  outline: none;
+  font-weight: 700;
+}
+
+.search::placeholder {
+  font-family: avenir, sans-serif;
+  color: #3d4e61;
+}
+
+.container {
+  margin: 20px 0;
+}
+
+.big__search {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  background: url(images/Search.svg) 25px center no-repeat;
+}
+
+.big__search:focus {
+  background: url(images/Search.svg) 25px center no-repeat;
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.small__search {
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  background: url(images/Search.svg) 12px 14px/11px no-repeat;
+}
+
+.small__search:focus {
+  background: url(images/Search.svg) 12px 14px/11px no-repeat ;
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
```

Comment:
if you have two or more similar elements with portions of similar styles with different values - use one of the elements as the basic case, and override necessary styles for other cases